### DISCUSSION TITLE
Where are the v10 docs archived?

### DISCUSSION THREAD

**Discussion by decipher-cs** [motiondivision/motion#2966] [Q&A] [answered]
I was looking for the docs for motion `v10.18.0` and I was wondering if they were just removed completely after `v11` launched. Usually old docs are kept under a subdomain, like how [react](https://18.react.dev/) and other websites do it, but that's clearly not the case here. Currently I was using web archive to reference the docs.

**Comment by mattgperry** [comment, marked as answer]
I added readmes to the functions with the biggest changes

https://github.com/motiondivision/motionone/tree/main/packages/dom/src/timeline

https://github.com/motiondivision/motionone/tree/main/packages/dom/src/animate
